frontend/src/component/operator-wizard/page-component/Page.js

```diff
@@ -248,6 +288,15 @@ export default class Page extends Component {
     return (
       <Form id={"form-page-" + this.props.pageNumber}>
         <div key={"page" + this.props.pageNumber}>
+          <Button
+            key="try"
+            variant="secondary"
```

Comment:
Is this a test? should it be merged?

Reply:
i will take the latest , please donot merge.